contracts/governance/FeeSharingCollector/FeeSharingCollector.sol

```diff
@@ -507,6 +548,56 @@ contract FeeSharingCollector is
         validFromCheckpointParam(_fromCheckpoint, msg.sender, RBTC_DUMMY_ADDRESS_FOR_CHECKPOINT)
         nonReentrant
     {
+        _withdrawRbtcTokenStartingFromCheckpoint(
+            address(0),
+            _fromCheckpoint,
+            _maxCheckpoints,
+            _receiver
+        );
```

Comment:
Here you withdraw all RBTC tokens, but only the native RBTC are withdrawn from the given _fromCheckpoint. This is not great. the other RBTC tokens might also have a lot of empty checkpoints and with this solution, they would all need to be claimed separately. 

I think it makes more sense to pass an array of _fromCheckpoints to `withdrawRBTCStartingFromCheckpoint` so that a starting checkpoint can be defined for each of the three assets.